src/components/selector-dropdown/__tests__/SelectorDropdown-test.js

```diff
@@ -423,26 +454,6 @@ describe('components/selector-dropdown/SelectorDropdown', () => {
         });
     });
 
-    describe('onItemClick', () => {
```

Comment:
This component doesn't have an `onItemClick()` function so I'm deleting this unit test.